examples_tests/61.BlitFilterTest/blit/formats/r11f_g11f_b10f.glsl

```diff
@@ -0,0 +1,59 @@
+#ifndef _NBL_GLSL_BLIT_R11FG11FB10F_INCLUDED_
+#define _NBL_GLSL_BLIT_R11FG11FB10F_INCLUDED_
+
+uint nbl_glsl_encode_ufloat(in float _f32, in uint mantissaMask, in uint mantissaBits)
+{
+	const uint f32 = floatBitsToUint(_f32);
+
+	if ((f32 & 0x80000000u) != 0u) // negative numbers converts to 0 (represented by all zeroes)
+		return 0;
+
+	const uint expMask = 0x1fu << mantissaBits;
+
+	const int exp = int(((f32 >> 23) & 0xffu) - 127);
+	const uint mantissa = f32 & 0x7fffffu;
+
+	uint f = 0u;
+	if (exp == 128) // inf / NaN
+	{
+		f = expMask;
+		if (mantissa != 0u)
+			f |= (mantissa & mantissaMask);
+	}
+	else if (exp > 15) // overflow, converts to infinity
+		f = expMask;
+	else if (exp > -15)
```

Comment:
you're making assumptions about the number of bits in the exponent

Reply:
I think this is irrelevant now.
Resolved.